integration-tests/tests/example-page.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { checkErrors } from '../support';
+
+const PLUGIN_TEMPLATE_NAME = 'console-plugin-template';
+const PLUGIN_TEMPLATE_PULL_SPEC = Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC');
+export const isLocalDevEnvironment =
+  Cypress.config('baseUrl').includes('localhost');
+
+const installHelmChart = () => {
+  cy.exec(
+    `cd ../../console-plugin-template && /tmp/helm upgrade -i ${PLUGIN_TEMPLATE_NAME} charts/openshift-console-plugin -n ${PLUGIN_TEMPLATE_NAME} --create-namespace --set plugin.image=${PLUGIN_TEMPLATE_PULL_SPEC}`,
+    {
+      failOnNonZeroExit: false,
+    },
+  ).then((result: any) => {
+    cy.visit(`/example`);
+    cy.log('Error: ', result.stderr);
+    cy.log('Success: ', result.stdout);
+  });
+};
+
+if (!Cypress.env('OPENSHIFT_CI') || Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC')) {
+  describe('Console plugin template test', () => {
+    before(() => {
+      cy.login();
+
+      if (!isLocalDevEnvironment) {
+        console.log('this is not a local env, installig helm');
+
+        cy.exec('cd ../../console-plugin-template && ./install_helm.sh', {
+          failOnNonZeroExit: false,
+        }).then((result) => {
+          cy.log('Error installing helm: ', result.stderr);
+          cy.log('Successfully installed helm: ', result.stdout);
+
+          installHelmChart();
+        });
+      } else {
+        console.log('this is a local env, not installing helm');
+
+        installHelmChart();
+      }
+    });
+
+    afterEach(() => {
+      checkErrors();
+    });
+
+    after(() => {
+      cy.logout();
```

Comment:
we should be also deleteing the project where the plugin lives
https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts#L163